### DISCUSSION TITLE
Do borderless / decorationless windows support resizing on wayland ?

### DISCUSSION THREAD

**Discussion by NovaliX-Dev** [rust-windowing/winit#3985] [General]
I'm on Fedora 41, KDE Plasma, with Wayland backend.

I'm trying to make a window which is resizable, but don't have decorations. Unfortunately i don't manage to achieve that goal.
I was wondering then if this "type" of window support resizing on that backend for the time being.

Thank you for your answers in advance.

**Comment by kchibisov** [comment]
you can implement resize yourself with drag_resize.
